v_4_2_0/master_template.go

```diff
@@ -264,7 +264,7 @@ systemd:
       --register-node=true \
       --register-with-taints=node-role.kubernetes.io/master=:NoSchedule \
       --kubeconfig=/etc/kubernetes/kubeconfig/kubelet.yaml \
-      --node-labels="node-role.kubernetes.io/master,role=master,ip=${DEFAULT_IPV4},{{.Cluster.Kubernetes.Kubelet.Labels}}" \
```

Comment:
Same as elsewhere: We rely on these old labels here and there so it makes sense to aling those as well. In `4.1.2` I added node selection for `role=master` to `calico-kube-controllers` in here. There might be other similar pinning as well.